dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -786,21 +786,21 @@ BEGIN
     IDS_ABOUT_VERSION_STRING "Versiune %S (%S)"
 
     /* context menus */
-    IDS_VIEW_LARGE "D&ale"
+    IDS_VIEW_LARGE "&Dale"
     IDS_VIEW_SMALL "&Pictograme"
     IDS_VIEW_LIST "&Listă"
     IDS_VIEW_DETAILS "&Detalii"
     IDS_SELECT "Selectea&ză"
     IDS_OPEN "&Deschide"
     IDS_CREATELINK "Creează sc&urtătură"
-    IDS_COPY "&Copiază"
-    IDS_DELETE "Șt&erge"
+    IDS_COPY "&Copiere"
+    IDS_DELETE "Ş&tergere"
     IDS_PROPERTIES "&Proprietăți"
-    IDS_CUT "&Decupează"
+    IDS_CUT "&Decupare"
     IDS_RESTORE "&Restabilește"
     IDS_FORMATDRIVE "F&ormatare…"
     IDS_RENAME "Rede&numește"
```

Comment:
At the top of the file you changed
MENUITEM "&Redenumește", IDM_RENAME
to
MENUITEM "Redenu&mire", IDM_RENAME

Therefore you have to change
IDS_RENAME "Rede&numește"
to
IDS_RENAME "Redenu&mire"
as well!

Reply:
Fixed! I stopped translating as I waited for your PR (and for others) related to shell32 in order to avoid conflicts.

Reply:
Confirmed. Resolved.